<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grillewindow</class>
 <widget class="QDialog" name="Grillewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètres</string>
  </property>
  <widget class="QDialogButtonBox" name="actionOk">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>318</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="verticalSpacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_form1">
      <property name="text">
       <string>Nombre de couleurs de rocailles maximum :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="donneeMaxColor">
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_fom2">
      <property name="text">
       <string>Enregistrer automatiquement une copie :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="donneeCopie">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_form3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Nombre de rocalles par rang :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="donneeSeedsInRow">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_path">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Prévisualisation
de l'image originale </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="actionOuvrir">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>431</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Choisir l'image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOk</sender>
   <signal>accepted()</signal>
   <receiver>Grillewindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOk</sender>
   <signal>rejected()</signal>
   <receiver>Grillewindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
